src/rtcModule/webrtc.cpp

```diff
@@ -4330,6 +4340,13 @@ void AudioLevelMonitor::OnData(const void *audio_data, int bits_per_sample, int
     }
 }
 
+bool AudioLevelMonitor::isDisabledAudioLevelMonitor() const
+{
```

Comment:
I think this method should rely exclusively on the new flag. It will be initialized upon the creation of the call and used directly (no need for this method) in order to decide whether the audio level should be monitored or not in `AudioLevelMonitor::OnData()`.

Reply:
Fixed